sql/core/src/test/scala/org/apache/spark/sql/MathFunctionsSuite.scala

```diff
@@ -258,6 +258,18 @@ class MathFunctionsSuite extends QueryTest with SharedSQLContext {
     )
   }
 
+  test("round/bround with table columns") {
```

Comment:
This is an end-to-end test for code-gen path. Could we add a unit test case in `MathExpressionsSuite`?

Reply:
All the tests in `MathExpressionsSuite` are testing about literals, I think it's better to improve `MathExpressionsSuite` for attributes in a new PR, instead of doing it in a folllow-up PR. BTW the original PR didn't add test in `MathExpressionsSuite` either.